.github/workflows/samples-python-petstore.yaml

```diff
@@ -0,0 +1,59 @@
+name: "Python Client: Petstore"
+
+on:
+  pull_request:
+    paths:
+      - samples/openapi3/client/petstore/python-aiohttp/**
+      - samples/openapi3/client/petstore/python/**
+      - .github/workflows/samples-python-petstore.yaml
+
+jobs:
+  build:
+    name: Test Python client
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          #- "3.11"
```

Comment:
may I know why 3.11 is skipped?

Reply:
There are some issues in the petstore tests, I will propose a fix so that we can enable 3.11 :+1: